src/redux/store.js

```diff
@@ -1,7 +1,27 @@
 import { createStore, combineReducers } from 'redux';
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { persistStore, persistReducer } from 'redux-persist';
 
-//import xx from './reducers';
+import countReducer from './reducers';
 
-const rootReducer = combineReducers({});
+const rootReducer = combineReducers({ countReducer: countReducer });
```

Comment:
```suggestion
const rootReducer = combineReducers({ count: countReducer });
```

This will let us refer to the updated time with `state.count.updated` - it felt to me like the object isn't really related to the reducer in the state, it is just created with the reducer